.github/CODEOWNERS

```diff
@@ -25,15 +25,15 @@ compaction/* @raphaelsc @nyh
 transport/*
```

Comment:
My question is for the line 15 (can't select it):

```
db/commitlog/* @elcallio
```

According to our meeting notes I think we should add @eliransin here?

Reply:
Indeed. I'll push a new version.